daemon/algod/api/server/v2/generated/routes.go

```diff
@@ -244,6 +316,16 @@ func (w *ServerInterfaceWrapper) ShutdownNode(ctx echo.Context) error {
 
 // GetStatus converts echo context to params.
 func (w *ServerInterfaceWrapper) GetStatus(ctx echo.Context) error {
+
+	validQueryParams := map[string]bool{}
+
+	// Check for unknown query parameters.
+	for name, _ := range ctx.QueryParams() {
+		if _, ok := validQueryParams[name]; !ok {
```

Comment:
In the empty case, the test is not requited.

Reply:
I would also like to move the maps to the top level so that they aren't recomputed every time. Perhaps the compiler is smart enough to do this automatically, but it wouldn't be too much work to scope the variables with the operation ID.

Another improvement is to skip these:
```
	// ------------- Optional query parameter "format" -------------
	if paramValue := ctx.QueryParam("format"); paramValue != "" {
	}
```

It seems to be an artifact related to required query parameters (of which we have none). There is an `else` generated which returns an error if the required parameter is missing. Alone the check is meaningless.

At the moment I'm not planning to add these improvements.